service/api_patch.go

```diff
@@ -155,14 +160,13 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		PatchString string `json:"patch"`
 		Githash     string `json:"githash"`
 		Message     string `json:"message"`
-		CommitQueue bool   `json:"commit_queue"`
 	}{}
 	if err = util.ReadJSONInto(util.NewRequestReader(r), &data); err != nil {
 		as.LoggedError(w, r, http.StatusBadRequest, err)
 		return
 	}
 
-	if p.Alias == evergreen.CommitQueueAlias && !data.CommitQueue {
+	if p.Alias == evergreen.CommitQueueAlias && !patch.IsMailboxDiff(data.PatchString) {
 		as.LoggedError(w, r, http.StatusBadRequest, errors.New("You may be using 'set-module' instead of 'commit-queue set-module', or your CLI may be out of date.\n"+
```

Comment:
I think we can be confident their CLI is up-to-date if they've created the commit queue patch in the first place.

Reply:
I think it's okay to leave this error just in case, they'll definitely know which one it is for them (if they create a patch, pause it, we deploy, and then they add to it, that'll hit this error).